iconservice/iconscore/icon_score_api_generator.py

```diff
@@ -79,8 +81,11 @@ def __generate_functions(src: list, score_funcs: list) -> None:
                     src.append(ScoreApiGenerator.__generate_normal_function(
                         func.__name__, is_readonly, is_payable, signature(func)))
                 elif func.__name__ == ScoreApiGenerator.__API_TYPE_FALLBACK:
-                    src.append(ScoreApiGenerator.__generate_fallback_function(
-                            func.__name__, is_payable, signature(func)))
+                    if func.__qualname__ != ScoreApiGenerator.__BASE_FALLBACK:
```

Comment:
1. `fallback` cannot be external by its definition, and this property has already been checked in external wrapper. So we can just check for the external flag in the first if statement.
2. `fallback` should be included in the api list only if it has `payable` decorator, and checking `is_payable` is enough since `IconScoreBase.fallback` doesn't have that flag.

The following code is my suggestion.
```
            try:
                if const_bit_flag & ConstBitFlag.External:
                    src.append(ScoreApiGenerator.__generate_normal_function(
                        func.__name__, is_readonly, is_payable, signature(func)))
                elif func.__name__ == ScoreApiGenerator.__API_TYPE_FALLBACK:
                    if is_payable:
                        src.append(ScoreApiGenerator.__generate_fallback_function(
                            func.__name__, is_payable, signature(func)))
            except ...:
```

Reply:
sugesstion 1 is good!
but I'm not sure about your second suggestion.
I wrote it below about that reason.

Reply:
disscusion fixed.
1. we don't allow non payable fallback. 
it can be existed only default function.
it would be impossible that you want to call fallback without ICX.

2. "method not found policy" is going to determine